tool/tctl/common/plugins_command.go

```diff
@@ -96,11 +127,80 @@ func (p *PluginsCommand) Cleanup(ctx context.Context, clusterAPI *auth.Client) e
 	return nil
 }
 
+func (p *PluginsCommand) InstallSCIM(ctx context.Context, client *auth.Client) error {
+	logrus.Debug("Fetching cluster info...")
+	info, err := client.Ping(ctx)
+	if err != nil {
+		logrus.Errorf("Failed fetching cluster info %v", err)
+		return trace.Wrap(err)
+	}
+
+	scimTokenHash, err := bcrypt.GenerateFromPassword([]byte(p.install.scim.token), bcrypt.DefaultCost)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	logrus.Debug("Validating SAML Connector ID...")
+	connector, err := client.GetSAMLConnector(ctx, p.install.scim.samlConnector, false)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	request := &pluginsv1.CreatePluginRequest{
+		Plugin: &types.PluginV1{
+			SubKind: types.PluginSubkindAccess,
```

Comment:
Do we need to be using this subkind? I think it was supposed to be used by access request plugins. Not sure what it affects.

Reply:
bump (i wasn't sure if this was addressed)

Reply:
It looks like the SubKind `types.PluginSubkindAccess` set for most of our plugins and this seems to be copy paste issue:
For instance OpenEngine sets the SubkindAccess: https://github.com/gravitational/teleport.e/blob/3907daa3a9db4bb5fc39a7605c32f551e2a021a6/lib/web/plugindescriptor.go#L173 


I don't see any condition based on plugin type so it should be safe to omit but @tcsc please confirm.

Reply:
Yeah, I've been digging around to find any consequences of using it, and running some tests. Can't see any consequences to changing it.